content/en/blog/_posts/2022-10-27-live-and-let-live-with-kluctl-and-ssa.md

```diff
@@ -0,0 +1,202 @@
+details about the conflicts. Kluctl can then use these details to figure out
+which fields are in conflict and which actors (field managers) have taken
+ownership of the conflicted fields.
+
+Then, for each field, Kluctl will decide if the conflict should be ignored or
+if it should be force-applied. If any field needs to be force-applied, Kluctl
+will retry the apply operation with the ignored fields omitted and the `force`
+flag being set on the API call.
+
+In case a conflict is ignored, Kluctl will issue a warning to the user so that
+the user can react properly (or ignore it forever...).
+
+That's basically it. That is all that is required to leverage SSA.
+Big thanks and thumbs-up to the Kubernetes developers who made this possible!
+
+## Conflict Resolution
+
+Kluctl has a few simple rules to figure out if a conflict should be ignored
+or force-applied.
+
+It first checks the field's actor (the field manager) against a list of known
+tools that are frequently used to perform manual modifications. These
+are for example `kubectl` and `k9s`. Any modifications performed with these tools
+are considered "temporary" and will be overwritten by Kluctl.
+
+If the field manager is not known by Kluctl, it will check if force-applying is
+requested for that field. Force-applying can be requested in different ways:
+
+1. By passing `--force-apply` to Kluctl. This will cause ALL fields to be force-applied on conflicts.
+2. By adding the [`kluctl.io/force-apply=true`](https://kluctl.io/docs/reference/deployments/annotations/all-resources/#kluctlioforce-apply) annotation to the object in question. This will cause all fields of that object to be force-applied on conflicts.
+3. By adding the [`kluctl.io/force-apply-field=my.json.path`](https://kluctl.io/docs/reference/deployments/annotations/all-resources/#kluctlioforce-apply-field) annotation to the object in question. This causes only fields matching the JSON path to be force-applied on conflicts.
+
+Marking a field to be force-applied is required whenever some other actor is
+known to erroneously claim fields (the ECK operator does this to the nodeSets
+field for example), you can ensure that Kluctl always overwrites these fields
+to the original or a new value.
+
+In the future, Kluctl will allow even more control about conflict resolution.
+For example, the CLI will allow to control force-applying on field level.
+
+## DevOps vs Controllers
+
+So how does SSA in Kluctl lead to "live and let live"?
+
+It allows the co-existence of classical pipelines (e.g. Github Actions or
+Gitlab CI), controllers (e.g. the HPA controller or GitOps style controllers)
+and even admins running deployments from their local machines.
+
+Wherever you are on your infrastructure automation journey, Kluctl has a place
+for you. From running deployments using a script on your PC, all the way to
+fully automated CI/CD with the pipelines themselves defined in code, Kluctl
```

Comment:
Remove `the` before `pipelines`

Reply:
@codablock your text as suggested is fine here.